bin/index.js

```diff
@@ -423,9 +438,16 @@ async function autoCmd(packageName, opts) {
 }
 
 async function cwdCmd(opts) {
-    const { depth: maxDepth = 4, output, nolock, full } = opts;
+    const { depth: maxDepth = 4, output, nolock, full, strategy } = opts;
+
+    if (strategy === VULN_MODE_NPM_AUDIT) {
+        vulnStrategy = VulnerabilityStrategy(NPMAuditStrategy());
```

Comment:
I think we need to add a method to dynamically generate the right strategy (with a clean switch). We will need to do this kind of initialization in several places.

This should prevent us from copying and pasting if/else everywhere

Reply:
I tried to move that into a function that should init the strategy and also its side effects which should be explicitly called from the outside (e.g : checkHydrateDB() should only be called internally by the SecurityWG strategy or by the hydrate-cmd)